src/main/java/com/iexec/worker/tee/TeeServicesPropertiesService.java

```diff
@@ -36,9 +36,8 @@ public TeeServicesPropertiesService(SmsService smsService,
         this.iexecHubService = iexecHubService;
```

Comment:
Changelog:
`* Remove useless casts.`
?

Reply:
I will keep remove unchecked casts instead of useless.
On the business code, the cast is done elsewhere and does not disappear.